### ISSUE TITLE
Open document settings has no visible focus state

### ISSUE THREAD

**Issue by joedolson** [WordPress/gutenberg#47662]
OK, I found that. It's...interesting. That button is contained inside `.edit-post-layout__toggle-sidebar-panel`,  which is positioned offscreen 9999em above the viewport. It has no visibility state at all. I'm guessing this was added at some point as a tool for screen readers only, as it's only even observable if you're using a screen reader. It's basically the same as the 'Open Publish Panel' control, but with no visible focus state. 

That control needs a visible focus state. 

![image](https://user-images.githubusercontent.com/1686451/216217215-fb5bce42-8837-426f-af35-03330561fd54.png)

Image shows the control with a manually adjusted position of `top:20em`.

_Originally posted by @joedolson in https://github.com/WordPress/gutenberg/issues/46851#issuecomment-1413066973_

**Comment by joedolson** [comment]
This needs handling pretty much the same as the 'Open Publish Panel' button.

**Comment by carolinan** [comment]
I may be missing something from the previous conversation.
I have Gutenberg trunk active. I am testing with a keyboard only, no screen reader, in Chrome, on Mac OS.

Open the block editor by creating a new post.

If the sidebar is closed:
1. The post title input is focused.
2. Press tab once: The first paragraph placeholder block is focused.
3. Press tab again: Here I can not tell what is focused. I do not see any visual indicator. This is confusing. Maybe the inserter button is focused?
4. Press tab again: There is a small panel visible in the bottom right corner with a button with the text Open document settings.

If the sidebar is not closed:
1. The post title input is focused.
2. Press tab once: The first paragraph placeholder block is focused.
3. Press tab again: Here I can not tell what is focused. I do not see any visual indicator. This is confusing. Maybe the inserter button is focused?
4. Press tab again: Focus is on the post tab (document tab) in the settings sidebar.
5. Press tab again: Focus is on the block settings tab.
6. Press tab again: Focus is on the close settings button.
7. Activate this button. Here I can not tell what is focused and I do not see any visual indicator. But I can see that the sidebar closed.
8. Press tab again. A panel with the button with the text "Open publish panel" is visible.
9. If I press tab now I end up in the title bar of the browser window.
10. But if I instead press shift + tab when I am on step 8, there is a small panel visible in the bottom right corner with a button with the text Open document settings.
11. And if I press shift + tab again from there, I end up on the last block in the content.
